### ISSUE TITLE
Error getting values of account in different currency

### ISSUE THREAD

**Issue by jen2014** [holtrop/ruby-gnucash#6]
I have a Euro account transferring amounts to a USD account:
`Assets:Current Assets:Checking Account:Euro
Assets:Current Assets:Checking Account:USD`
The gem accurately lists the Euro transactions, but does not get the values converted in USD that correctly appear in the Gnucash application.  The Euro values are shown instead.
I have attached the test gnucash book, as well as the ruby script used to test it.
It seems the gem has a problem accurately interpreting the splits in that type of gnucash usage.

[gct_s.rb.zip](https://github.com/holtrop/ruby-gnucash/files/1251840/gct_s.rb.zip)
[test_book.zip](https://github.com/holtrop/ruby-gnucash/files/1251845/test_book.zip)

**Comment by holtrop** [comment]
I think I fixed this properly - let me know if not. Pushed v1.2.1.

**Comment by jen2014** [comment]
I checked it on a small number of accounts, and it works so far. Thanks a lot.
It also partially fixes a related issue I was going to mention: when you deal with Stock or Mutual Fund accounts, you prefer obtaining ting for each transaction the number of shares and the balance in number of shares (as displayed in GnuCash).  Your fix now gets the number of shares (the 'quantity' item in the XML file) but the balance is a dollar amount, not a quantity of shares.  I have tried a modification in  gnucash 1.2.0 which adds a new variable @balance_shares to deal with it. It worked, but I need to verify it in 1.2.1 and learn how to share it on Github.
